io_scene_nif/modules/object/object_export.py

```diff
@@ -267,18 +260,20 @@ def export_node(self, b_obj, space, parent_block, node_name):
                 else:
                     node.flags = 0x000C  # morrowind
 
-        self.set_object_matrix(b_obj, space, node)
+        self.set_object_matrix(b_obj, node)
 
         if b_obj:
             # export animation
             if b_obj_ipo:
-                if any(b_obj_ipo[b_channel] for b_channel in (Ipo.OB_LOCX, Ipo.OB_ROTX, Ipo.OB_SCALEX)):
-                    self.nif_export.animationhelper.export_keyframes(b_obj_ipo, space, node)
-                self.export_object_vis_controller(b_obj, node)
+				# no anim support here for now!
+                pass
+                # if any(b_obj_ipo[b_channel] for b_channel in (Ipo.OB_LOCX, Ipo.OB_ROTX, Ipo.OB_SCALEX)):
```

Comment:
I would uncomment and add a TODO so that we remember to come back and fix this.

Reply:
Uncommented and fixed object anim export.
export_object_vis_controller() remains commented out for the time being as it needs to be updated to 2.7 API.